rust/src/bin/lq.rs

```diff
@@ -47,7 +47,8 @@ async fn main() {
             let dataset = Dataset::open(uri).await.unwrap();
             let mut scanner = dataset.scan().unwrap();
             scanner.limit(*n, None);
-            println!("{:?}", scanner.next_batch().await.unwrap().unwrap());
+            let batch = scanner.next_batch().await.unwrap().unwrap();
+            println!("Batch: {:?}", batch);
```

Comment:
Be fun to do like a "next pointer" that you can feed back into lq to keep going through batches. Not need now ofc